src/npctalk.cpp

```diff
@@ -1352,6 +1352,27 @@ void dialogue::add_topic( const talk_topic &topic )
     topic_stack.push_back( topic );
 }
 
+talker *dialogue::actor( const bool is_beta ) const
+{
+    if( ( is_beta && !has_beta ) || ( !is_beta && !has_alpha ) ) {
+        debugmsg( string_format( "Tried to use an invalid %s talker.", is_beta ? "beta" : "alpha" ) );
+    }
+    return ( is_beta ? beta : alpha ).get();
+}
+
+dialogue::dialogue( std::unique_ptr<talker> alpha_in,
+                    std::unique_ptr<talker> beta_in )
+{
+    has_alpha = alpha_in != nullptr;
+    has_beta = beta_in != nullptr;
+    if( has_alpha ) {
+        alpha = std::move( alpha_in );
+    }
+    if( has_beta ) {
+        beta = std::move( beta_in );
+    }
```

Comment:
```suggestion
    }
    if( !has_alpha && !has_beta ) {
        debugmsg( "Constructed a dialogue with no actors!" );
    }
```

Reply:
We may also be able to use a static assert to promote this to a compile time error.